pkg/types/baremetal/defaults/platform.go

```diff
@@ -0,0 +1,67 @@
+package defaults
+
+import (
+	"fmt"
+	"net"
+
+	"github.com/openshift/installer/pkg/types"
+	"github.com/openshift/installer/pkg/types/baremetal"
+)
+
+// Defaults for the baremetal platform.
+const (
+	LibvirtURI         = "qemu:///system"
```

Comment:
the default is different on the `libvirt` platform. why is this different?

Reply:
`qemu:///system` is the [appropriate way to talk to localhost's system mode daemon](https://libvirt.org/uri.html#URI_qemu).  Libvirt platform appears to assume [libvirt is available in 192.168.122.1, and prefers to use TCP](https://github.com/openshift/installer/pull/1873/files/377d79c71997710f7e22400ff7f6d4fa2f6ed8d8..9355096cb9b3a59ee3dd35987f82443fc315054b#diff-02cb3da236b573e9e866932657f6463a). I do not know why, but I don't think that is a good default for the baremetal platform.  We cannot assume a user is using the libvirt default network, or that they use 192.168.122.0/24.

Reply:
libvirt used qemu:///system until we started trying to launch workers via libvirt. Then we just hard coded the 192 address (which mad a lot of people unset/unusable).

We probably 'could' force people to remove all security from their bootstrap node (as we do for libvirt), but I for one strongly prefer this.

Reply:
i'm fine with the default, but things like ^^ are helpful to include the commit message.